src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1826,59 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+
+    public function updateVerificationStatus($userToken,$verificationStatus){
+        if (empty($userToken)) {
+            throw new HyperwalletArgumentException('userToken is required!');
+        }
+        if (empty($verificationStatus)) {
+            throw new HyperwalletArgumentException('verification status is required!');
+        }
+        if($verificationStatus != User::VERIFICATION_STATUS_REQUESTED){
+            throw new HyperwalletArgumentException("Expected verification status is REQUESTED!");
+        }
+        $user = $this->getUser($userToken);
+        $fromValidationStatus = $user->getVerificationStatus();
+        $expectedFromValues = array(User::VERIFICATION_STATUS_REQUIRED,User::VERIFICATION_STATUS_NOT_REQUIRED);
+        if(!in_array($fromValidationStatus,$expectedFromValues)){
+            throw new HyperwalletException("The from verification status is expected to be 'Required' or 'Not required'");
+        }
+        $user = new User(array('verificationStatus'=> User::VERIFICATION_STATUS_REQUESTED));
+        $responseUser = $this->client->doPut('/rest/v3/users/{user-token}', array('user-token' => $userToken), $user, array());
+        return $responseUser;
```

Comment:
Do we need to convert the responseUser to User object here?

Reply:
Converted the return value to an user object and ran the IT scenarios.